features/restaurant_grouping.feature

```diff
@@ -0,0 +1,20 @@
+Feature: User can select restaurant by category
+  As a user
+  In order to get choose what to eat
+  I would like to see a restaurant by category
+
+  Background:
+    Given the following category exists
+      | name    | description  |
+      | Thai    | Thai food    |
+      | Mexican | Mexican food |
+    And the following restaurants exist
+      | name       | address     |
+      | My Thai    | Stockholm   |
+      | My Mexican | Mexico city |
```

Comment:
You assign the category in the Factory. That's okay. To be in control wat category each restaurant belongs to, you can add that to the table and rewrite your step definition. Find the RestaurantCategory by name before you create the restaurant and assign it as a association

Reply:
Was trying to have the factory assign it, but it s not working properly since it always creates the object that its being associated with, I am scrapping this and assign it to the category in the steps file.